python/ray/tests/test_dataclient_disconnect.py

```diff
@@ -1,57 +1,29 @@
-from ray.util.client.ray_client_helpers import ray_start_client_server
-from unittest.mock import Mock
+from ray.util.client.ray_client_helpers import ray_start_client_server_pair
+import time
 import pytest
 
 
-def test_dataclient_disconnect_on_request():
-    with ray_start_client_server() as ray:
-        assert ray.is_connected()
-
-        @ray.remote
-        def f():
-            return 42
-
-        assert ray.get(f.remote()) == 42
-        # Force grpc to error by sending garbage request
-        with pytest.raises(ConnectionError):
-            ray.worker.data_client._blocking_send(Mock())
-
-        # Client should be disconnected
-        assert not ray.is_connected()
-
-        # Test that a new connection can be made
-        connection_data = ray.connect("localhost:50051")
-        assert connection_data["num_clients"] == 1
-        assert ray.get(f.remote()) == 42
-
-
 def test_dataclient_disconnect_before_request():
```

Comment:
Same as previous comment.

Reply:
Restored this test and the previous one, just needed some adjustements.